config/checkstyle_non_main_files_suppressions.xml

```diff
@@ -104,4 +104,8 @@
     <suppress checks="RegexpSingleline"
               files="src[\\/]test[\\/]resources[\\/].*[\\/]checks[\\/]metrics[\\/]classfanoutcomplexity[\\/]InputClassFanOutComplexityPackageName.*\.java"/>
 
+    <!-- to suppress archunit ViolationStore and Rules files -->
+    <suppress checks="NewlineAtEndOfFile" files="[\\/]archunit_store"/>
+    <suppress checks="RegexpSingleline" files="[\\/]archunit_store"/>
```

Comment:
remove this folder from target of validation - https://github.com/checkstyle/checkstyle/blob/master/config/ant-phase-verify.xml#L34 .
Please try to configure archunit plugin to make such folder in `target` folder.

Reply:
Done